modules/stanford_profile_styles/lib/scss/base/stanford_profile_styles.scss

```diff
@@ -50,6 +50,27 @@ svg {
   }
 }
 
+// Hide the SVG in localist widgets
+.localist-widget {
+  a {
+    &.su-link--external {
+      &::after {
+        @include hide-visually;
+      }
+    }
+  }
+}
+
+#lclst_widget_footer {
```

Comment:
What happens when you have 2 widgets on the same page?

Reply:
The id isn't going to change; that's coming from inside the embed content, even if there's more than one.  It's bad code from localist.

Reply:
Yeah, I just validated that by placing two widgets on the same page, and it works as expected.

![image](https://user-images.githubusercontent.com/1263892/144647142-000778b5-0e38-4f96-9956-5b041f444486.png)

Reply:
I tried it in VoiceOver and the link was read as an External Link. I remember @cjwest  was concerned about that.

Reply:
Ok thanks for checking. That is bad code from Localist but what  can ya do.